backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/modelvalidator/M_ShipmentSchedule.java

```diff
@@ -235,6 +240,29 @@ public void invalidateSchedulesWithOldAndNewHeaderAggregationKey(final I_M_Shipm
 		invalidSchedulesInvalidator.flagHeaderAggregationKeysForRecompute(headerAggregationKeys);
 	}
 
+	@ModelChange(
+			timings = { ModelValidator.TYPE_AFTER_CHANGE},
+			ifColumnsChanged = { I_M_ShipmentSchedule.COLUMNNAME_IsClosed})
+	public void updateIsDeliveryClosed(@NonNull final I_M_ShipmentSchedule shipmentSchedule)
+	{
+		final I_C_OrderLine orderLine = shipmentSchedule.getC_OrderLine();
```

Comment:
@cristinamghita  cc @metas-ts ,
leave it like this for now, but for the future:

avoid model getters, use DAO/repo/BL to fetch the order line